### ISSUE TITLE
Equations in README are unreadable on dark background

### ISSUE THREAD

**Issue by dpsanders** [stevengj/cubature#21]
The equations in the README are unreadable on a dark background.

I believe it should now be possible to just use inline LaTeX instead of images.

**Comment by stevengj** [comment]
Yes, feel free to submit a PR to use inline LaTeX for these equations.

**Comment by ararslan** [comment]
I've submitted a PR for this: #22
